src/components/MessagesList/MessagesGroup/Message/MessagePart/Reactions.vue

```diff
@@ -274,20 +276,24 @@ export default {
 </script>
 <style lang="scss" scoped>
 .reactions-wrapper {
+	--minimal-button-width: 48px;
```

Comment:
Can it be calculated?

Reply:
if we can do it for width of `emoji character`, `whitespace`, `digit 1-9`, and add button paddings. Otherwise I don't see any options to do it =/